src/main/scala/zio/redis/options/shared.scala

```diff
@@ -0,0 +1,5 @@
+package zio.redis.options
+
+trait shared extends zset {
```

Comment:
You can use capitalized names for these, they're all subsumed by `options` anyhow. Also, no need to extend `zset` here (done in `options`).

Reply:
As discussed in our chat, we'll get rid of `shared` all together.